xml/System.Windows/FontWeights.xml

```diff
@@ -37,7 +37,7 @@
 |<xref:System.Windows.FontWeights.Black%2A><br /><br /> <xref:System.Windows.FontWeights.Heavy%2A>|900|  
 |<xref:System.Windows.FontWeights.ExtraBlack%2A><br /><br /> <xref:System.Windows.FontWeights.UltraBlack%2A>|950|  
   
- For more information about the [!INCLUDE[TLA#tla_opentype](~/includes/tlasharptla-opentype-md.md)] `usWeightClass` value, see the ["Font File Tables"](http://www.microsoft.com/typography/otspec/otff.htm) section of the [!INCLUDE[TLA2#tla_opentype](~/includes/tla2sharptla-opentype-md.md)] specification.  
+ For more information about the [!INCLUDE[TLA#tla_opentype](~/includes/tlasharptla-opentype-md.md)] `usWeightClass` value, see the ["Font File Tables"](https://docs.microsoft.com/en-us/typography/opentype/spec/otff) section of the [!INCLUDE[TLA2#tla_opentype](~/includes/tla2sharptla-opentype-md.md)] specification.  
```

Comment:
I'd replace the last part of the sentence with

`see the [usWeightClass](https://docs.microsoft.com/typography/opentype/spec/os2#usweightclass) section of the [!INCLUDE[TLA2#tla_opentype](~/includes/tla2sharptla-opentype-md.md)] specification.`

Reply:
Hi, I changed the last part of the sentence as requested, please check